src/core.ts

```diff
@@ -1139,8 +1139,27 @@ function applyHeadersMut(targetHeaders: Headers, newHeaders: Headers): void {
 }
 
 export function debug(action: string, ...args: any[]) {
+  const sensitiveHeaders = ["authorization", "api-key"];
```

Comment:
nit: would be better to define this outside the function and as a `Set` imo

```suggestion
const SENSITIVE_HEADERS = new Set(["authorization", "api-key"]);

export function debug(action: string, ...args: any[]) {
```

Reply:
Thank you. I updated based on the suggestion